examples/Jenks.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bb0c3ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install jenkspy"
```

Comment:
Or, better yet, add `jenkspy` to our `requirement-dev.txt` to ensure it will be available in our dev environment. This environment is used to test the notebooks and the tests should pass with it.

Reply:
Yes also that